src/mmw/sass/components/_modals.scss

```diff
@@ -138,3 +138,22 @@
   top: 1rem;
   left: 2rem;
 }
+
+.modal-popup {
+    .fa {
+      font-size: 1.0rem;
+      padding: 0 0.1rem 0 0.1rem;
+    }
+
+    .fa-info-circle {
+      color: green;
```

Comment:
Looks fine in this case, but does bring me back to this comment from @jfrankl 
https://github.com/azavea/pwd-parcel-viewer/issues/359#issuecomment-250157013